python_modules/dagster/dagster/_daemon/sensor.py

```diff
@@ -863,13 +876,33 @@ def submit_run_request(run_request) -> SubmitRunRequestResult:
             f"Skipping {skipped_count} {'run' if skipped_count == 1 else 'runs'} for sensor "
             f"{external_sensor.name} already completed with run keys: {seven.json.dumps(run_keys)}"
         )
+    yield
```

Comment:
somewhat confused by this change and how it relates to the other stuff -- certainly can be convinced that it was redundant though

Reply:
if a sensor yields and RunRequest that is a single run and a RunRequest that is a backfill, we dont want to set the tick status to SUCCESS or SKIPPED until both run requests have been processed. So this section of code got moved up here https://github.com/dagster-io/dagster/pull/22681/files#diff-d1be406eec832cf545bf2c1c2f5e2e33316184c7c0f5250d128f326811592935R683-R686

Reply:
ah makes sense!